followthemoney/schema/Message.yaml

```diff
@@ -0,0 +1,71 @@
+Message:
+  extends:
+    - Interval
+    - Folder
+    - PlainText
+    - HyperText
+  label: Message
+  plural: Messages
+  matchable: false
+  generated: true
+  featured:
+    - subject
+    - threadTopic
+    - date
+    - sender
+    - recipients
+  caption:
+    - subject
+    - threadTopic
+    - title
+    - fileName
+  properties:
+    subject:
+      label: Subject
+      type: string
+    threadTopic:
+      label: Thread topic
+      type: string
+    sender:
+      label: "Sender"
+      type: entity
+      range: LegalEntity
+      reverse:
+        name: messagesSent
+        label: "Messages sent"
+    senderAccount:
+      label: "Sender Account"
+      type: entity
+      range: UserAccount
+      reverse:
+        name: messagesSent
+        label: "Messages sent"
```

Comment:
Don't know how to deal with this. I'm basically duplicating the same data here for `messagesSent` which is not ideal.